### DISCUSSION TITLE
请问大家，现在是否有不需要第三方的情况下实现两方安全计算矩阵乘积的工作？

### DISCUSSION THREAD

**Discussion by denghenggang** [secretflow/secretflow#1525] [General]
感谢指路

**Comment by lanyy9527** [comment]
可以基于SPU秘密共享的方式来实现两方安全计算，可参考：https://www.secretflow.org.cn/zh-CN/docs/secretflow/v1.10.0b0/tutorial/spu_basics

**Comment by Stu-Yang** [comment]
在[issue#1037基于隐语的安全两方计算：支持的运算、协议和指标测试](https://github.com/secretflow/secretflow/issues/1037)中有写一个安全两方矩阵乘法的demo测试，即`matmul`，可以参考一下

PS: 可能需要检查一下所述方法是否利用可信第三方来实现离线阶段乘法三元组生成
